docs/proposals/operator-sdk-v1-migration.md

```diff
@@ -0,0 +1,285 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: crd-domain-rename
+authors:
+  - @adambkaplan
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-12-08
+last-updated: 2020-12-09
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Operator SDK v1 Migration
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions
+
+1. Do we declare the migration to SDK v1 a breaking update? Do we not support upgrades from v0.2.x
+   to v0.3.0?
+2. Should we move our base domain to `shipwright.io`?
+3. Should we use this opportunity to improve the namings of our components. Ex - rename
+   `build-operator` to `build-controller-manager`
```

Comment:
👍

Reply:
"controller-manager" is common terminology for "a binary that runs one or more related controllers." It's used in Kubernetes for the core APIs, and the sdk defaults use "controller-manager" to describe the generated deployment.

Reply:
yes but Build is a single controller, BuildRun also, right?